packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -182,6 +184,8 @@ async function handleSuccessResponse(
   controller: AbortController
 ) {
   const logger: Logger = response.locals.logger
+  const keyVersion: number = Number(response.header('keyVersion'))
+  logger.info({ keyVersion }, 'Signer responded with keyVersion')
```

Comment:
If the key is not the version the combiner is expecting / prepared to accept, we should probably discard the response. The risk here is that including signature into the BLS aggregation will result in the whole batch being corrupted. Even if the client can recover from it, which I am not sure we've implemented, it will waste a lot of time.